src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -156,10 +158,16 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
   }
 
   const handleCancel: HandleCancelCallback = async () => {
-    orderTracking.clickedCancelExpressCheckout({
-      order: orderData,
-      paymentMethod: expressCheckoutType as string,
-    })
+    if (!errorRef.current) {
+      orderTracking.clickedCancelExpressCheckout({
+        order: orderData,
+        paymentMethod: expressCheckoutType as string,
+      })
+    }
+
+    if (errorRef.current) {
+      shippingContext.actions.dialog.showErrorDialog()
+    }
```

Comment:
1. Should we set errorRef back to null before rejecting completely?
2. Should we use the string in the error dialog and/or track it?